cmd/serve/dlna/cds.go

```diff
@@ -130,6 +130,20 @@ func (cds *contentDirectoryService) readContainer(o object, host string) (ret []
 		return
 	}
 
+	// if there's a "Subs" child directory, add its children to the list as well,
+	// so mediaWithResources is able to find them.
+	for _, node := range dirEntries {
+		if node.Name() == "Subs" && node.IsDir() {
```

Comment:
Does this directory name need to be configured?

It certainly needs documenting though - a line in the docs about here maybe?

https://github.com/rclone/rclone/blob/70e8ad456fcc627f57c001aa82ac925bd9fb47a8/cmd/serve/dlna/dlna.go#L50

Reply:
They didn't mention the subtitle discovery yet at all, but yes, can add.

`Subs` is a convention I've seen so far quite recently, and so far nothing else seems to be commonly used. I don't think it makes a lot of sense to make this configurable.

Reply:
Added a commit documenting the functionality.